### PR TITLE
Feature/initial repo setup

### PR DESCRIPTION
**Pull request by nshumoogum**
### What

Initial gubbings for repository:

* ci set up
* Update README.md
* `.gitignore`, `LICENSE.md` and `CONTIBUTING.md` files updated/created

### How to review

Check general setup of repository is not missing anything.

### Who can review

Team B

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+### Configuration
+
+| Environment variable       | Default                              | Description
+| -------------------------- | -------------------------------------| -----------
+| BIND_ADDR                  | :22900                               | The host and port to bind to
+| HIERARCHY_API_URL          | http://localhost:22600               | The host name for the Hierarchy API
+| DATASET_API_URL            | http://localhost:22000               | The host name for the Dataset API
+| KAFKA_ADDR                 | localhost:9092                       | A list of Kafka host addresses
+| CONSUMER_GROUP             | dp-search-builder                    | The name of the Kafka consumer group
+| CONSUMER_TOPIC             | hierarhy-built                       | The name of the topic to consumes messages from
```
Comment (CarlHembrough) [inline README.md]: typo in `hierarchy`

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
-# dp-search-builder
\ No newline at end of file
+dp-search-builder
+==================
+
+### Getting started
+
+* Run ```brew install kafka```
```
Comment (CarlHembrough) [inline README.md]: Consider referring to the dp-import docs for common infrastructure setup, and having only app specifics here.

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+
+### Healthcheck
+
+The endpoint `/healthcheck` checks the connection to the database and returns
```
Comment (CarlHembrough) [inline README.md]: Take a look at the health check 'framework' classes added to go-ns when adding a healthcheck.

**Review by CarlHembrough — CHANGES_REQUESTED** [review]
Looks good. See the suggestions, and also remember the following (not that it's part of this PR):
```
How do I add a repo to Concourse CI?

 - add CI scripts to your repo under CI directory
   - Example : https://github.com/ONSdigital/dp-import-api/tree/develop/ci

 - add you repo details to this file
   - https://github.com/ONSdigital/dp-ci/blob/develop/pipelines/branch-manager/applications.yml

How do I get my repo to deploy via Concourse CI?

 - extend the repo details to include an environment mapping
   - https://github.com/ONSdigital/dp-ci/blob/develop/pipelines/branch-manager/applications.yml

 - add nomad script to your repo
    - example: https://github.com/ONSdigital/dp-import-api/blob/develop/dp-import-api.nomad

 - add nomad job resource directory + files for your repo
    - https://github.com/ONSdigital/dp-ci/tree/develop/nomad-job-resource-defs

- add encrypted secrets file under the appropriate environment directory
    - https://github.com/ONSdigital/dp-ci/tree/develop/secrets

- add vault policy to dp-setup repo
    - ensure your on the right branch - for CMD: https://github.com/ONSdigital/dp-setup/tree/aws-develop%23cmd
    - Add entry for your repo in this file: https://github.com/ONSdigital/dp-setup/blob/aws-develop%23cmd/terraform/vault-policies/policies.tf
    - add a new .hcl file for your repo here: https://github.com/ONSdigital/dp-setup/tree/aws-develop%23cmd/terraform/vault-policies/policies
    - add an entry for your repo here: https://github.com/ONSdigital/dp-setup/blob/aws-develop%23cmd/terraform/vault-policies/policies/awdry.hcl

```

File: README.md (hunk 2 above)
Comment (nshumoogum) [reply README.md]: 👍

**Review by CarlHembrough — APPROVED** [review]
👍  - one thing I forgot to mention is the ports doc: https://github.com/ONSdigital/dp/blob/master/PORTS.md

**Comment by nshumoogum** [comment]
Added search builder port to `https://github.com/ONSdigital/dp/blob/master/PORTS.md`; see https://github.com/ONSdigital/dp/pull/36

**Pull request merged**